main.nf

```diff
@@ -0,0 +1,403 @@
+log.debug ""
+
+log.info "[Git Info]"
+log.info "$workflow.repository - $workflow.revision [$workflow.commitId]"
+log.info ""
+
+log.info "Options"
+log.info "======="
+log.info ""
+log.info "[Atlas]"
+log.info "Atlas Anatomy: $params.atlas_anat"
+log.info "Atlas Directory: $params.atlas_directory"
+log.info ""
+log.info "[Priors options]"
+log.info "BS Tracking Mask: $params.use_bs_tracking_mask"
+log.info "BS Endpoints Mask: $params.use_bs_endpoints_include"
+log.info "Endpoints Mask Dilation: $params.bs_endpoints_mask_dilation"
+log.info "Seeding From Tracking Mask: $params.use_tracking_mask_as_seeding"
+log.info "Tracking Mask Dilation: $params.bs_tracking_mask_dilation"
+log.info ""
+log.info "[Tracking options]"
+log.info "Local Tracking: $params.local_tracking"
+log.info "PFT Tracking: $params.pft_tracking"
+log.info "Algo: $params.algo"
+log.info "Seeding Type: $params.seeding"
+log.info "Number of Seeds: $params.nbr_seeds"
+log.info "Random Seed: $params.tracking_seed"
+log.info "Minimum Length: $params.min_length"
+log.info "Maximum Length: $params.max_length"
+log.info "Compressing Threshold: $params.compress_error_tolerance"
+log.info ""
+log.info "[Recobundles options]"
+log.info "Segmentation with Recobundles: $params.recobundles"
+log.info "Whole Brain Clustering Threshold: $params.wb_clustering_thr"
+log.info "Model Clustering Threshold: $params.model_clustering_thr"
+log.info "Prunning Threshold: $params.prunning_thr"
+log.info "Outlier Removal Alpha: $params.outlier_alpha"
+log.info ""
+log.info ""
+
+log.info "Input: $params.root"
+root = file(params.root)
+/* Watch out, files are ordered alphabetically in channel */
+    in_data = Channel
+        .fromFilePairs("$root/**/{*fa.nii.gz,*fodf.nii.gz,*tracking_mask.nii.gz}",
+                        size: 3,
+                        maxDepth:2,
+                        flat: true) {it.parent.name}
+
+    map_pft = Channel
+        .fromFilePairs("$root/**/{*map_exclude.nii.gz,*map_include.nii.gz}",
+                        size: 2,
+                        maxDepth:2,
+                        flat: true) {it.parent.name}
+
+    atlas_anat = Channel.fromPath("$params.atlas_anat")
+    atlas_bundles = Channel.fromPath("$params.atlas_directory/*.trk")
+    algo_list = params.algo?.tokenize(',')
+
+(anat_for_registration, anat_for_deformation, fod_and_mask_for_priors) = in_data
+    .map{sid, anat, fodf, tracking_mask -> 
+        [tuple(sid, anat),
+        tuple(sid, anat),
+        tuple(sid, fodf, tracking_mask)]}
+    .separate(3)
+
+(map_in_for_tracking, map_ex_for_tracking) = map_pft
+    .map{sid, map_exclude , map_include -> 
+        [tuple(sid, map_include),
+        tuple(sid, map_exclude)]}
+    .separate(2)
+
+workflow.onComplete {
+    log.info "Pipeline completed at: $workflow.complete"
+    log.info "Execution status: ${ workflow.success ? 'OK' : 'failed' }"
+    log.info "Execution duration: $workflow.duration"
+}
+
+anat_for_registration
+    .combine(atlas_anat)
```

Comment:
Please use concat.

Reply:
Concat doesn't work. Did you mean to simply replace combine with concat or is there something else to do.
For context, anat_for_registration is all of my anat and atlas_anat is a single file channel